.eslintrc.js

```diff
@@ -55,4 +55,8 @@ module.exports = {
     es6: true,
     browser: true,
   },
+  ignorePatterns: [
+    // ignore old Actor example
+    '**/example_/*.js',
+  ],
```

Comment:
We should probably fix those old example lint wise...

Reply:
These look to be old examples. See: [Actor](https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/Actor/example_/index.js), [BoundingBox](https://github.com/Kitware/vtk-js/blob/master/Sources/Common/DataModel/BoundingBox/example_/index.js). How about we remove them instead, or move example usage code into `api.md`?

Reply:
Remove the actor but the boundingBox one should be replace to a test if not already the case. In other words, they should be removed but we should make sure we have a test for BoundingBox